### PR TITLE
Tensor-src-tizensensor: complete initial implementation. @open sesame 12/05 13:32

### PR DESCRIPTION
**Pull request by myungjoo**
All the required callbacks are implemented.
    
Next step: write test cases.
    
This addresses #1143
    
Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>

### CODE REVIEW

File: ext/nnstreamer/tensor_source/tensor_src_tizensensor.c [hunk 1]
```diff
@@ -1032,21 +1053,215 @@ static GstFlowReturn
...
+              event.value_count, self->src_spec.value_count));
+      retval = GST_FLOW_ERROR;
+      goto exit_unmap;
+    }
+
+    /* 2. Write timestamp/duration info to buffer */
+
+    /* 2-1. Find out the delay already occured */
+    clock_gettime(CLOCK_MONOTONIC, &t);
+    ts = (unsigned long long)(t.tv_sec) * 1000000LL +
```
Comment (jaeyun-jung) [inline ext/nnstreamer/tensor_source/tensor_src_tizensensor.c]: maybe same with g_get_monotonic_time()

Comment (myungjoo) [reply ext/nnstreamer/tensor_source/tensor_src_tizensensor.c]: Applied!

File: ext/nnstreamer/tensor_source/tensor_src_tizensensor.c [hunk 2]
```diff
@@ -898,18 +910,22 @@ _ts_get_gstcaps_from_conf (GstTensorSrcTIZENSENSOR *self)
...
+    g_assert (typestr && typestr[0]);
+    tensor = g_strdup_printf("other/tensor, dimension=%u:%u:%u:%u, type=%s ; "
+        "other/tensors, num_tensors=1, dimensions=%u:%u:%u:%u, type=%s",
         spec->tinfo.dimension[0], spec->tinfo.dimension[1],
-        spec->tinfo.dimension[2], spec->tinfo.dimension[3],
+        spec->tinfo.dimension[2], spec->tinfo.dimension[3], typestr,
         spec->tinfo.dimension[0], spec->tinfo.dimension[1],
-        spec->tinfo.dimension[2], spec->tinfo.dimension[3]);
+        spec->tinfo.dimension[2], spec->tinfo.dimension[3], typestr);
+    g_free (typestr);
```
Comment (jaeyun-jung) [inline ext/nnstreamer/tensor_source/tensor_src_tizensensor.c]: gst_tensor_get_type_string() does not return newly allocated string

Comment (myungjoo) [reply ext/nnstreamer/tensor_source/tensor_src_tizensensor.c]: Applied. Thanks!

**Pull request merged**